torch/csrc/lazy/core/ir.h

```diff
@@ -138,6 +133,22 @@ class TORCH_API Node {
 
   virtual std::string ToString() const;
 
+ protected:
```

Comment:
what's the significance of this change to protected?  since we still default to not using 'reuse' mode, i guess it doesn't change any actual behavior (e.g. prohibiting creating nodes).